.github/ISSUE_TEMPLATE/bug.yml

```
@@ -33,7 +33,18 @@ body:
     id: steps-to-reproduce
     attributes:
       label: Steps To Reproduce
-      placeholder: Please write the steps in a list form
+      placeholder: |
+        Please write the steps in a list form i.e.:
```

https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

```suggestion
        Please write the steps in a list form e.g.:
```